wp-parsely.php

```diff
@@ -22,20 +22,29 @@
  * Requires WP:       4.0.0
  */
 
-require 'src/class-parsely.php';
+// If this file is called directly, abort.
+if ( ! defined( 'WPINC' ) ) {
+	die;
+}
 
 if ( class_exists( 'Parsely' ) ) {
-	define( 'PARSELY_VERSION', Parsely::VERSION );
-	if ( ! defined( 'PARSELY_PLUGIN_DIR' ) ) {
-		define( 'PARSELY_PLUGIN_DIR', plugin_dir_path( __FILE__ ) );
-	}
-	if ( ! defined( 'PARSELY_PLUGIN_URL' ) ) {
-		define( 'PARSELY_PLUGIN_URL', plugin_dir_url( __FILE__ ) );
-	}
-	$parsely = new Parsely();
+	return;
+}
+
+if ( ! defined( 'PARSELY_PLUGIN_DIR' ) ) {
+	define( 'PARSELY_PLUGIN_DIR', plugin_dir_path( __FILE__ ) );
+}
+if ( ! defined( 'PARSELY_PLUGIN_URL' ) ) {
+	define( 'PARSELY_PLUGIN_URL', plugin_dir_url( __FILE__ ) );
 }
 
-require 'src/class-parsely-recommended-widget.php';
+require PARSELY_PLUGIN_DIR . 'src/class-parsely.php';
+
+define( 'PARSELY_VERSION', Parsely::VERSION );
```

Comment:
Instead of having a class constant, let's move the canonical value to this root file, where it's easier to find, and change along with the value in the plugin headers.

Alternatively, don't define this constant, but use the class constant as the canonical value. That may change as we break down the `Parsely` class though, which means any references would have to change.

Reply:
> Instead of having a class constant, let's move the canonical value to this root file, where it's easier to find, and change along with the value in the plugin headers.

I dig it.  Will follow up with a PR immediately.

Reply:
Incorporated the version constant change into #266.